discovery/triton/triton.go

```diff
@@ -187,6 +191,10 @@ func (d *Discovery) refresh() (tg *targetgroup.Group, err error) {
 	}()
 
 	var endpoint = fmt.Sprintf("https://%s:%d/v%d/discover", d.sdConfig.Endpoint, d.sdConfig.Port, d.sdConfig.Version)
+	if d.sdConfig.Groups != nil {
+		endpoint = fmt.Sprintf("%s?groups=%s", endpoint, strings.Join(d.sdConfig.Groups, ","))
```

Comment:
Is there any escaping needed here?

Reply:
I added escaping anyway, so this should be fixed.